doc/api/child_process.markdown

```diff
@@ -200,14 +200,12 @@ spawned directly as a new process making it slightly more efficient than
 [`child_process.exec()`][]. The same options are support by both
 `child_process.exec()` and `child_process.execFile()`.
 
-    const exec = require('child_process').execFile;
-    const child = execFile('cat *.js bad_file | wc -l',
-      (error, stdout, stderr) => {
-        console.log(`stdout: ${stdout}`);
-        console.log(`stderr: ${stderr}`);
-        if (error !== null) {
-          console.log(`exec error: ${error}`);
-        }
+    const execFile = require('child_process').execFile;
+    const child = execFile('/bin/cat', ['/etc/paths'], (error, stdout, stderr) => {
```

Comment:
Why not call an executable that has a better chance of existing on a system with node installed:
`('node', ['-p', 'process.version'], ...`